sample/src/main/java/io/mosaicnetworks/sample/Message.java

```diff
@@ -162,6 +166,7 @@ public Date getCreatedAt() {
      */
     @Override
     public byte[] toBytes() {
-        return gson.toJson(this).getBytes();
+        String json = gson.toJson(this);
```

Comment:
Any reason to exapnd this? Or was there a log statement inserted there in the meantime that has been removed?

Reply:
yep.. log statement